portal/portal-sprint-101.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: /release-notes/portal/sprint-101
+title: Sprint 101 (May 31, 2021)
+---
+
+# fiskaltrust.Portal - Sprint 101
+_May 31, 2021_
+
+**Usability improvements**
+
+In this sprint, we have introduced several usability improvements including improving the overall performance.
+
+## Features
+
+### Middleware Configuration
+- [Visual improvements to Middleware component dialogs](#visual-improvements-to-middleware-component-dialogs)
+- [Feedback on Cashbox rebuild](#feedback-on-cashbox-rebuild)
+
+## Middleware Configuration
+
+### Visual improvements to Middleware component dialogues
+
+Some minor visual improvements of the TableViews in Portal (DE) have been implemented. We invested some time in improving the way our tables are displayed. Visually nothing major should have changed, aside from a smaller screen size -in which the user is navigating the TableViews page- does not affect anymore the way the tables are shown. The performance of the page in different resolutions has also been improved.
```

Comment:
Those Middleware configuration page updates are actually not DE specific.

Reply:
Also small point: Search improved.
As we no longer include the object structure in the Search filter.
Previous:
Searchterm: Nam -> Testing: MyObject**Nam**e: Hubert -> Hubert would show up in the search results
Now:
SearchTerm: Nam -> Testing: Hubert -> no longer added to the results